migrations/migrations/20210317205208-user-roles.js

```diff
@@ -8,5 +8,9 @@ module.exports = {
       allowNull: false,
       defaultValue: 'regular'
     }),
-  down: queryInterface => queryInterface.removeColumn('user', 'role')
+  down: queryInterface =>
+    Promise.all([
```

Comment:
Are you modifying an existing migration? What if you need to rollback? I think that is a better idea to generate a new migration to do that.

Reply:
This modifications was because I forgot to put it at the time of creation. It is related to go back process.
This allow us to delete the `enum_user_role` when use `npm run migrations:undo`.